ruby-app-onbuild/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ruby-app:2.1.5
+
+# SSH proxy settings
+ENV SSH_AUTH_SOCK /tmp/ssh-auth
+ENV SSH_AUTH_PROXY_PORT 1234
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+ONBUILD COPY Gemfile /usr/src/app/
+ONBUILD COPY Gemfile.lock /usr/src/app/
+
+# Hack to install private gems
+ONBUILD RUN socat UNIX-LISTEN:$SSH_AUTH_SOCK TCP4:$(ip route|awk '/default/ {print $3}'):$SSH_AUTH_PROXY_PORT & bundle install
+
+ONBUILD COPY . /usr/src/app
+ONBUILD RUN mkdir /usr/src/app/public/assets
```

Comment:
This assets dir probably belongs in the ruby-webapp-onbuild, not here – if we assume this image is going to be used for non-rails things then we def don't need it here.